src/APL/vercel-apl.ts

```diff
@@ -122,4 +122,24 @@ export class VercelAPL implements APL {
     }
     return [authData];
   }
+
+  // eslint-disable-next-line class-methods-use-this
+  async isReady(): Promise<AplReadyResult> {
+    for (const requiredEnvKey of Object.values(VercelAPLVariables)) {
+      const envValue = process.env[requiredEnvKey];
+
+      if (!envValue || envValue.length === 0) {
+        return {
+          ready: false,
+          error: new Error(
+            `Env variable ${requiredEnvKey} doesnt exist or is empty. Ensure env variables exist`
+          ),
+        };
+      }
+    }
+
+    return {
+      ready: true,
+    };
+  }
```

Comment:
We can display all of them at once to avoid debugging variables one by one.

```suggestion
  async isReady(): Promise<AplReadyResult> {
    const missingVariables: string[] = [];
    for (const requiredEnvKey of Object.values(VercelAPLVariables)) {
      const envValue = process.env[requiredEnvKey];

      if (!envValue || envValue.length === 0) {
        missingVariables.push(requiredEnvKey)
      }
    }
    
    if (missingVariables.length > 0) {
        return {
          ready: false,
          error: new Error(
            `Env variables ${missingVariables} doesn't exist or is empty. Ensure env variables exist`
          ),
        };
      }

    return {
      ready: true,
    };
  }
```

Reply:
Good, I added custom error with stored errors (raw array) + message, so it can be easily parsed